VMBackup/setup.py

```diff
@@ -188,6 +202,7 @@ def copy(src, dst):
 copybinary(workloadutils_folder, final_workloadutils_path)
 final_plugin_conf_path = final_folder_path + '/main'
 copybinary(binary_entry, final_binary_path)
+copybinary(binary_entry, final_Arm64binary_path)
```

Comment:
is the same binary being copied to both folders ?

Reply:
good catch....missed updating source path